kafka-reactive-system/src/main/scala/org/patricknoir/kafka/reactive/client/KafkaRClient.scala

```diff
@@ -37,6 +38,16 @@ class KafkaReactiveClient(settings: KafkaRClientSettings)(implicit system: Actor
    * @return
    */
   //TODO: I'm messing with Error vs Throwable => replace Error with Throwable!
-  def request[In: ReactiveSerializer, Out: ReactiveDeserializer](destination: String, payload: In)(implicit timeout: Timeout): Future[Error Xor Out] =
-    (kafkaClientService ? KafkaRequest(destination, serialize(payload), timeout, settings.inboundResponseQueue, implicitly[ReactiveDeserializer[Out]])).mapTo[Error Xor Out]
+  def request[In: ReactiveSerializer, Out: ReactiveDeserializer](destination: String, payload: In)(implicit timeout: Timeout): Future[Error Xor Out] = {
+    destination match {
```

Comment:
We should handle also the case the destination match fails, I believe this will cause a runtime exception rather than a Failure[Error] ? Maybe we can just add another case _ => Future.failure(....)